pagopa/io-app-design-system
# [IOAPPFD0-140] Remove `PictogramSection` from the codebase

## Short description
This PR removes the `PictogramSection` from the codebase

To learn more about this removal, please refer to the following PR:
- https://github.com/pagopa/io-app/pull/4944

## How to test
Run the example app and go to the `Pictograms` page. You shouldn't see a `PictogramSection` block.
